README.md

```diff
@@ -108,6 +108,32 @@ module.exports = {
 }
 ```
 
+### Disabling HMR
+
+By default Hot Module Replacement is disabled in following situations:
+
+ * Webpack `target` is `node`
+ * Webpack doesn't minify the code 
+ * `process.env.NODE_ENV !== 'production'`
```

Comment:
I actually wrote the conditions when the HMR is disabled...

The code is:

```js
const shouldHotReload = isHmrEnabled && !isServer && !this.minimize &&
  process.env.NODE_ENV !== 'production'
```

So `shouldHotReload === false` if `isServer === true`, `this.minimize === true` or `process.env.NODE_ENV === 'production'`